catalog/core/catalog-core-solrcommands/src/test/java/org/codice/ddf/commands/solr/TestBackupCommand.java

```diff
@@ -142,6 +199,296 @@ public void testSystemPropertiesNotSet() throws Exception {
         backupCommand.doExecute();
     }
 
+    @Test
+    public void testPerformSolrCloudSynchronousBackup() throws Exception {
+
+        // Set the solr client type system property so that the
+        // BackupCommand knows that it needs to backup solr cloud.
+        setupSolrClientType(SOLR_CLOUD_CLIENT_TYPE);
+
+        // Setup BackupCommand
+        BackupCommand backupCommand = new BackupCommand() {
+            @Override
+            SolrClient getSolrClient() {
+                return miniSolrCloud.getSolrClient();
+            }
+
+            // We get the solr client from the MiniSolrCloudCluster, so we don't
+            // want to shut it down after each test. We don't create a MiniSolrCloudCluster
+            // for each test to reduce the time it takes to run the tests.
```

Comment:
👍  nice comment. Is there a way to re-use `BackupCommand()` so you don't have to define it in each method? It seems like we could reduce a lot of code here if we were able to do that, but you may have already looked at the alternatives and came up with this approach.